config/config_test.go

```diff
@@ -53,18 +53,29 @@ func (s *RemoteParamOperatorTestSuite) SetupTest() {
 
 func (s *RemoteParamOperatorTestSuite) TearDownTest() {
 	_ = s.appBase.GetStore().Close()
+
+	_ = os.Setenv("REMOTE_PARAM_DOMAIN_DATA", "")
+	_ = os.Setenv("REMOTE_PARAM_RESOURCE_DATA", "")
 }
 
 func (s *RemoteParamOperatorTestSuite) TestSetRemoteParams_operatorIsNil() {
 	s.remoteParamOperator.EXPECT().LoadParameter("/chainbridge/fee-oracle/domainData").Times(0)
 	s.NotPanics(func() { s.conf.SetRemoteParams(nil) }, "should not panic")
 }
 
-func (s *RemoteParamOperatorTestSuite) TestSetRemoteParams_Failure() {
+func (s *RemoteParamOperatorTestSuite) TestSetRemoteParams_nonExistingParam_Failure() {
 	s.remoteParamOperator.EXPECT().LoadParameter("/chainbridge/fee-oracle/domainData").Return(nil, errors.New("error"))
+
+	_ = os.Setenv("REMOTE_PARAM_DOMAIN_DATA", "/chainbridge/fee-oracle/domainData")
```

Comment:
should I preserve the former parameter using the old naming convention in order to not break this test?

Reply:
@rbadillap no need, this is unit test